manifests/server/database.pp

```diff
@@ -85,7 +89,16 @@
       $database_suffix = ''
     }
 
-    $subname = "//${database_host}:${database_port}/${database_name}${database_suffix}"
+    $subname_default = "//${database_host}:${database_port}/${database_name}${database_suffix}"
+
+    if $postgresql_ssl_on {
+      $subname = "${subname_default}?\
```

Comment:
This seems to be creating large spaces in the conf file, after the first space it seems to stop reading any data.